### ISSUE TITLE
Some rules don't work

### ISSUE THREAD

**Issue by ChaseMoskal** [palantir/tslint#3344]
Hi folks, I'm having a rough time trying to imagine why I can't get some simple rules to work:

**I can't get these rules to do anything:**
- `no-irregular-whitespace`
- any of the `whitespace` rules

However, in the same tsconfig and ts sources, these other rules are being enforced as expected.
- `semicolon`
- `no-trailing-whitespace`
- `trailing-comma`

### Bug Report

- __TSLint version__: `5.7.0`
- __TypeScript version__: `2.5.3`
- __Running TSLint via__: `node`

#### TypeScript code being linted

```ts
const x= {

	a: 1,


	b: 2
}
```

with `tslint.json` configuration:

```json
{
	"rules": {
		"no-irregular-whitespace": true,
		"whitespace": {
			"check-branch": true,
			"check-decl": true,
			"check-operator": true,
			"check-module": true,
			"check-separator": true,
			"check-rest-spread": true,
			"check-type": true,
			"check-typecast": true,
			"check-type-operator": true,
			"check-preblock": true
		}
	}
}
```

#### Actual behavior

Code passes linting without error or warning.

#### Expected behavior

I'm expecting two linter failures:
1. "no-irregular-whitespace" should bark about the excess newlines between `a` and `b`.
2. "whitespace" should bark about the goofed whitespace before the assignment operator `=`.

**Comment by ajafff** [comment]
1. `no-irregular-whitespace` only warns about irregular whitespace *characters* like vertical tab, non breaking space, and so on.
Use `no-consecutive-blank-lines` for your use case.

2. that's probably a bug or not implemented for certain cases.

**Comment by ajafff** [comment]
Closing, because 2. is already tracked by #3199

**Comment by ChaseMoskal** [comment]
@ajafff — Thanks for the info, `no-consecutive-blank-lines` is surely what I was really looking for :+1: 

However, I noticed that the issue #3199 doesn't actually cover my complaint against whitespace check-decl — specifically, the title of the issue you cited:

> *whitespace rule 'check-decl' fails **when statement does not include a variable declaration keyword***

My example 2 ***does*** include a variable declaration keyword — so this is not a duplicate issue.

I'll go ahead and mention this on that issue.
